packages/status-js/vite.config.ts

```diff
@@ -14,7 +14,7 @@ const external = [
 export default defineConfig(({ mode }) => {
   const alias: Alias[] = []
 
-  if (mode === 'test') {
+  if (process.env.VITEST === 'true' || mode === 'test') {
```

Comment:
I thought `Vitest` sets mode to `test`, did this change?

Reply:
It does, but I added this because I wasn't always using Vitest, but `vite-node`.